src/server/ws-routes/controllers/Notifications.js

```diff
@@ -0,0 +1,58 @@
+const namespaceInfo = require('../../protocol/protocol.json').notifications
+const EVENT_TYPES = namespaceInfo.eventTypes
+
+/**
+ *  /Notifications websocket namespace
+ * @namespace
+ */
+class Notifications {
+	constructor({
+		UserModel = require('../../models/user'),
+		utils = require('../utilities')
+	} = {}) {
+		this.UserModel = UserModel
+		this.utils = utils
+	}
+
+	/**
+	 * Remove notifications
+	 * @name removeNotifications
+	 * @memberof Notifications
+	 * @member
+	 * @property {object[]} notificationIds Array of all ids of notifications you want to remove
```

Comment:
Isn't it `string[]`?

Reply:
structure looks like this : 
[
    {
        _id: 0987654365768908976875
    },
    {
         _id:9876543567890876546
    }
]

Reply:
Why underscore tho? Underscore in JS world usually means that this property is "private".